tests/test_flows.py

```
@@ -83,6 +86,110 @@ def __init__(self):
         self.tcpdump_proc = None
         self.bridge_name = 'br-lan'
 
+    def get_bridge_mac(self):
+        inspect = json.loads(subprocess.check_output(('docker', 'network', 'inspect',
+                                                      'prplMesh-net-{}'.format(self.opts.unique_id))))
+        prplmesh_net = inspect[0]
+        # podman adds a 'plugins' indirection that docker doesn't have.
+        if 'plugins' in prplmesh_net:
+            bridge = prplmesh_net['plugins'][0]['bridge']
+        else:
+            # docker doesn't report the interface name of the bridge. So format it based on the
+            # ID.
+            bridge_id = prplmesh_net['Id']
+            bridge = 'br-' + bridge_id[:12]
+        return bridge
+
+    def start_tshark(self, bridge, duration=None, reset=False):
+        self.tshark_instances = {}
+        self.tshark_capture = {}
+        if reset:
+            for t in self.tshark_instances:
+                try:
+                    t.captured_packets = []
+                finally:
```

What's the point in this clause?
if there's an exception in the try clause it will still crush the program. 
If you want to ignore exceptions you should use `except` and not `finally`.